tools/sapp/sapp/interactive.py

```diff
@@ -1409,7 +1412,8 @@ def _create_issue_output_string(
                 f" Sources: {sources_output if sources_output else 'No sources'}",
                 f"   Sinks: {sinks_output if sinks_output else 'No sinks'}",
                 (f"Location: {issue.filename}" f":{issue.location}"),
-            ]
+                f"Min_trace_length_to_sources: {issue.min_trace_length_to_sources}",
```

Comment:
Do you mind making the formatting match the above lines (ex. aligned colons, human readable). Maybe like this:
```
            [
                f"Issue {issue.id}",
                f"            Code: {issue.code}",
                f"         Message: {issue.message}",
                f"        Callable: {issue.callable}",
                f"         Sources: {sources_output if sources_output else 'No sources'}",
                f"           Sinks: {sinks_output if sinks_output else 'No sinks'}",
                f"        Location: {issue.filename}:{issue.location}",
                f"Min Trace Length: Sink ({issue.min_trace_length_to_sources}) | Source ({issue.min_trace_length_to_sinks})",            ]
```
I'm not attached to exactly the format I suggested, I'm just trying to make it visually consistent

Reply:
yes, I like the suggested formatting. I'll update the code. The output will look like this:
```
Issue 1
            Code: 5001
         Message: Possible shell injection Data from [UserSpecified] source(s) may reach [RemoteCodeExecution] sink(s)
        Callable: source.convert
         Sources: UserSpecified
           Sinks: RemoteCodeExecution
        Location: source.py:9|22|32
Min Trace Length: Sink (0) | Source (1)
```